lib/response.js

```diff
@@ -805,6 +805,7 @@ res.get = function(field){
 
 res.clearCookie = function clearCookie(name, options) {
   var opts = merge({ expires: new Date(1), path: '/' }, options);
+  delete opts.maxAge;
```

Comment:
```suggestion
  var opts = merge({ path: '/' }, options);
  // Force cookie expiration by setting `expires` to a past date
  opts.expires = new Date(1);
  // Ensure `maxAge` is explicitly removed
  delete opts.maxAge
```
Let's do this, to also ensure they cannot overwrite `expires` via the options object either

Reply:
Hold off on this until we get clarity on https://github.com/expressjs/express/issues/4851#issuecomment-2094505498 however

Reply:
we can't land this in v4, so will need to target the v5 dev branch